src/panel-rooms.tsx

```diff
@@ -166,9 +166,10 @@ class RoomsPanel extends PSRoomPanel {
 	}
 	renderRoomList(title: string, rooms?: RoomInfo[]) {
 		if (!rooms || !rooms.length) return null;
+		const sortedRooms = rooms.sort((a, b) => /* Ascending order */ (b.userCount || 0) - (a.userCount || 0));
```

Comment:
Hm, I think it looks better on multiple lines.

```suggestion
		const sortedRooms = rooms.sort((a, b) =>
			// Ascending order
			(b.userCount || 0) - (a.userCount || 0)
		);
```

Reply:
Possible alternative?

```suggestion
		// Ascending order
		const sortedRooms = rooms.sort((a, b) => (b.userCount || 0) - (a.userCount || 0));
```

Reply:
Also wait you mean "Descending order", right? Most to least users?

Reply:
I prefer the second one. I don't think it's long/complex enough to warrant more than one line.

> Also wait you mean "Descending order", right? Most to least users?

 I forgot how English works I guess, lol.